src/Core/StringExtensions.cs

```diff
@@ -145,7 +145,7 @@ public static int IndexOfAny(this string s, string[] words, StringComparison com
 
         public static string FixExtraSpaces(this string s)
         {
-            if (string.IsNullOrEmpty(s) || s.Trim().Length == 0)
+            if (string.IsNullOrWhiteSpace(s))
```

Comment:
```
PS> [string]::IsNullOrEmpty('')
True
```
[MSDN](https://msdn.microsoft.com/en-us/library/system.string.isnullorwhitespace): “true if the value parameter is null or String.Empty, or if value consists exclusively of white-space characters.”